### ISSUE TITLE
Label 'Line Thickness' is not associated with it's edit combo box.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1672]
Repro steps:
Open the Application Calculator
Navigate to ‘Graphing’ button and activate it.
Now navigate to the 'Share' button and and activate it.
Observe the issue.

Actual Result:
Label 'Line Thickness' is not associated with it's combo box. Screen reader announce "Extra line width combo-box collapsed".

Expected Result:
Label 'Line Thickness' should be associated with it's combo box. Screen reader should announce "Line Thickness Extra line width selected combo-box collapsed".
